File: src/test/java/easy/MajorityElementTest.java
package easy;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class MajorityElementTest {

    @Test
    void majorityElement() {
        assertEquals(3, new MajorityElement().majorityElement(new int[]{3,2,3}));
        assertEquals(2, new MajorityElement().majorityElement(new int[]{2,2,1,1,1,2,2}));
    }
}

File: src/test/kotlin/MajorityElementTest.kt
import leetcode.easy.MajorityElement
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class MajorityElementTest {
    @Test
    fun `testing majority element`(){
        assertEquals(3, MajorityElement().majorityElement(intArrayOf(3,2,3)))
    }
}